client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,52 +1,44 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit, startsWith, endsWith } from 'lodash';
 
-module.exports = React.createClass( {
-	displayName: 'SharingButtonsPreviewAction',
+const SharingButtonsPreviewAction = ( props ) => {
+	const { active, position, icon, children } = props;
+	const classes = classNames( 'sharing-buttons-preview-action', {
+		'is-active': active,
+		'is-top': startsWith( position, 'top' ),
+		'is-right': endsWith( position, '-right' ),
+		'is-bottom': startsWith( position, 'bottom' ),
+		'is-left': endsWith( position, '-left' )
+	} );
 
-	propTypes: {
-		active: React.PropTypes.bool,
-		position: React.PropTypes.oneOf( [
-			'top-left',
-			'top-right',
-			'bottom-left',
-			'bottom-right'
-		] ),
-		icon: React.PropTypes.string,
-		onClick: React.PropTypes.func
-	},
+	return (
+		<button type="button" className={ classes } { ...omit( props, [ 'active', 'position' ] ) }>
```

Comment:
It would maybe be an overkill but at least I would know what I can pass down to that component (ie "oh, I can pass down [xyz] prop? nice!"). Maybe we could at least include it in the readme.

Anyways, too small detail to block this PR :D